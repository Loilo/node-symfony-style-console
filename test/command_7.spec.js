

const wrapper = require('./wrapper')

test('07 Ensure questions do not output anything when input is non-interactive', wrapper(io => {
    io.input.setInteractive(false)

    io.title('Title')
    io.askHidden('Hidden question')
    io.choice('Choice question with default', ['choice1', 'choice2'], 'choice1')
    io.confirm('Confirmation with yes default', true)
    io.text('Duis aute irure dolor in reprehenderit in voluptate velit esse')
}))
