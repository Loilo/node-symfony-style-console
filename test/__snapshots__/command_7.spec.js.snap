// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`07 Ensure questions do not output anything when input is non-interactive 1`] = `
"Title
=====

 Duis aute irure dolor in reprehenderit in voluptate velit esse"
`;
